draft-ietf-quic-tls.md

```diff
@@ -1127,7 +1149,9 @@ The header protection key is not updated.
 
 If the packet can be decrypted and authenticated using the updated key and IV,
 then the keys the endpoint uses for packet protection are also updated.  The
-next packet sent by the endpoint will then use the new keys.
+next packet sent by the endpoint MUST then use the new keys.  Once an endpoint
+has sent a packet encrypted with a given key phase, it MUST NOT send a packet
+encrypted with a strictly older key phase.
```

Comment:
Why strictly older? Just older should be sufficient.

Reply:
Fixed